entc/cmd/entc/entc.go

```diff
@@ -133,6 +133,7 @@ func loadGraph(path string, cfg gen.Config) (*gen.Graph, error) {
 	}
 	cfg.Schema = spec.PkgPath
 	cfg.Package = filepath.Dir(spec.PkgPath)
```

Comment:
From the `filepath godoc:
filepath.Dir:
> Dir calls Clean on the path and trailing slashes are removed.
filepath.Clean
> Finally, any occurrences of slash are replaced by Separator.

And `Separator` is set to `os.PathSeparator`. Can you try replace `filepath` with `path`? 
```suggestion
	cfg.Package = path.Dir(spec.PkgPath)
```

Reply:
OK, checked it now on Windows machine.

```go
println(path.Dir("a/b/c"))     // prints "a/b"
println(filepath.Dir("a/b/c")) // prints "a\b"
```